tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/prep-abort-test.go

```diff
@@ -33,81 +31,39 @@ var _ = Describe(
-			})
-
-			By("Deleting prep cgu created on target hub cluster", func() {
-				err := cnfhelper.DeleteIbuTestCguOnTargetHub(cnfinittools.TargetHubAPIClient, tsparams.PrepCguName,
-					tsparams.IbuCguNamespace)
-				Expect(err).ToNot(HaveOccurred(), "Failed to delete prep cgu on target hub cluster")
-			})
-
-			By("Deleting finalize cgu created on target hub cluster", func() {
-				err := cnfhelper.DeleteIbuTestCguOnTargetHub(cnfinittools.TargetHubAPIClient, tsparams.FinalizeCguName,
-					tsparams.IbuCguNamespace)
-				Expect(err).ToNot(HaveOccurred(), "Failed to delete finalize cgu on target hub cluster")
+			By("Deleting IBGU on target hub cluster", func() {
+				_, err := newIbguBuilder.DeleteAndWait(1 * time.Minute)
+				Expect(err).ToNot(HaveOccurred(), "Failed to delete IBGU on target hub cluster")
 			})
 
 			// Sleep for 10 seconds to allow talm to reconcile state.
-			// Sometimes if the next test re-creates the CGUs too quickly,
+			// Sometimes if the next test re-creates the IBGUs too quickly,
 			// the policies compliance status is not updated correctly.
 			time.Sleep(10 * time.Second)
 		})
 
 		It("Upgrade prep abort flow", reportxml.ID("68956"), func() {
-			By("Creating, enabling ibu pre-prep CGU and waiting for CGU status to report completed", func() {
-				prePrepCguBuilder := cgu.NewCguBuilder(cnfinittools.TargetHubAPIClient,
-					tsparams.PrePrepCguName, tsparams.IbuCguNamespace, 1).
-					WithCluster(tsparams.TargetSnoClusterName).
-					WithManagedPolicy(tsparams.PrePrepPolicyName).
-					WithCanary(tsparams.TargetSnoClusterName)
-				prePrepCguBuilder.Definition.Spec.Enable = ptr.To(true)
-
-				prePrepCguBuilder, err := prePrepCguBuilder.Create()
-				Expect(err).ToNot(HaveOccurred(), "Failed to create pre-prep CGU.")
-
-				_, err = prePrepCguBuilder.WaitUntilComplete(10 * time.Minute)
-				Expect(err).ToNot(HaveOccurred(), "Pre-prep CGU did not complete in time.")
-			})
 
-			By("Creating, enabling ibu prep CGU and waiting for CGU status to report completed", func() {
-				prepCguBuilder := cgu.NewCguBuilder(cnfinittools.TargetHubAPIClient,
-					tsparams.PrepCguName, tsparams.IbuCguNamespace, 1).
-					WithCluster(tsparams.TargetSnoClusterName).
-					WithManagedPolicy(tsparams.PrepPolicyName).
-					WithCanary(tsparams.TargetSnoClusterName)
-				prepCguBuilder.Definition.Spec.Enable = ptr.To(true)
+			By("Creating IBGU and monitoring IBU status to report completed", func() {
 
-				prepCguBuilder, err := prepCguBuilder.Create()
-				Expect(err).ToNot(HaveOccurred(), "Failed to create prep CGU.")
+				newIbguBuilder = ibgu.NewIbguBuilder(cnfinittools.TargetHubAPIClient,
+					tsparams.IbguName, tsparams.IbguNamespace).
+					WithClusterLabelSelectors(tsparams.ClusterLabelSelector).
+					WithOadpContent("oadp-cm", "ztp-group").
+					WithSeedImageRef("registry.kni-qe-18.lab.eng.tlv2.redhat.com:5000/ibu/seed:4.17.0-rc.1", "4.17.0-rc.1").
```

Comment:
I think we need to load the seed image/version and oadp cm name/namespace as config variables as in https://github.com/openshift-kni/eco-gotests/pull/227/files#diff-6278a8b3de0a7acdaf56dda60e1d478bccfe97bfe808d762c352e5d3716ec039 ?

Reply:
Updated- thank you.